src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -316,6 +324,17 @@ data class IndividualUser(
   override fun canSetTestClock(): Boolean {
     return userType == UserType.SuperAdmin
   }
+
+  override fun canSetWithdrawalUser(accessionId: AccessionId): Boolean {
```

Comment:
Should this have userId semantics as well.. since contributors can assign themselves..?

Reply:
My mental model here was the Unix `chown` command -- as a non-superuser you can create files and they'll be owned by your user ID, but that doesn't mean you have permission to set the ownership of files.

If there's another name for this that would get that idea across better, I could definitely rename it.

Reply:
Thanks for the clarification, this name works just fine.